sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -2296,6 +2296,13 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
     }
   }
 
+  /**
+   * Create a [[ShowDatabasesStatement]] command.
+   */
+  override def visitShowDatabases(ctx: ShowDatabasesContext): LogicalPlan = withOrigin(ctx) {
```

Comment:
Should these classes be `ShowNamespacesStatement`, `ShowNamespaces`, and `ShowNamespacesExec` instead?

Reply:
Changed to Namespace*